### ISSUE TITLE
Tags that will be broken if Tiny Skeletons is not installed

### ISSUE THREAD

**Issue by Mechalopa** [Uraneptus/SnowPig-forge-1.16#7]
```
{
  "replace": false,
  "values": [
    "minecraft:stray",
    {
      "id": "tinyskeletons:baby_stray",
      "required": false
    }
  ]
}
```
I think `killer_for_snowpig_disc.json` needs to be like this in case Tiny Skeletons is not installed.

**Comment by Uraneptus** [comment]
Oh yeah you're right, thank you! ^^

**Comment by Uraneptus** [comment]
Fixed in 9cbdb00
